### PR TITLE
🚮 Make amp-story version 0.1 invalid

### PR DESCRIPTION
**Pull request by newmuis**
Additionally, this fixes a few examples which did not specify amp-story metadata.

We'd want for this to go live 2019-03-19 ideally (as that is what we have [communicated](https://groups.google.com/a/ampproject.org/forum/#!topic/amp-story-origin-trial/FzW3YR5E6sQ)), but it's technically okay if it happens slightly _after_ that, just not before.

Intent to implement (deprecate): #21251

### CODE REVIEW

File: extensions/amp-story/1.0/test/validator-amp-story-animations-error.out [hunk 1]
```diff
@@ -75,4 +74,4 @@ amp-story/1.0/test/validator-amp-story-animations-error.html:54:12 The attribute
 |        </amp-story-page>
 |    </amp-story>
 |  </body>
-|  </html>
+|  </html>
```
Comment (Enriqe) [inline extensions/amp-story/1.0/test/validator-amp-story-animations-error.out]: nit: newlines here and in other `.out` files. (--update-tests flag doesn't add it but you can add it manually)

File: extensions/amp-story/0.1/test/validator-amp-story-animations-error.out [hunk 2]
```diff
@@ -75,4 +83,4 @@ amp-story/0.1/test/validator-amp-story-animations-error.html:54:12 The attribute
 |        </amp-story-page>
 |    </amp-story>
 |  </body>
-|  </html>
+|  </html>
```
Comment (gmajoulet) [inline extensions/amp-story/0.1/test/validator-amp-story-animations-error.out]: Nit: newlines

**Comment by Enriqe** [comment]
image

**Comment by newmuis** [comment]
PTAL @Gregable @Enriqe @gmajoulet 

This has been marinating for a while; see #21251

**Comment by newmuis** [comment]
Ping @Gregable

**Comment by newmuis** [comment]
PTAL @Gregable; having 0.1 stories be valid can actually break AMP viewer functionality for stories, since 0.1 stories do not support the functionality that viewers expect.

**Pull request merged**